landing-page/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+  <head>
+    <style>
+      body {
+  background-color: MistyRose;
+}
+      </style>
+  </head>
+  <body>
+      <style>
+          h1{
```

Comment:
These styles can all be included in the same style element that you already have nested under the `head` element.